### PR TITLE
Simplify update workflow

### PR DESCRIPTION
**Pull request by colin-rogers-dbt**
Follow on PR to address issues with creating the dev release.

### CODE REVIEW

File: test/unit/github_client/test_get_release.py [hunk 1]
```diff
@@ -21,3 +22,31 @@ def test_get_latest_bundle_release_gets_latest_release(monkeypatch, input_versio
     input_version = Version.coerce(input_versions)
     latest_version, is_draft, latest_release = get_latest_bundle_release(str(input_version))
     assert str(latest_version) == output_versions
+
+
+def test_get_bundle_release_returns_none_if_no_release_found(monkeypatch):
+    mock_version_1_1 = Mock(tag_name="1.1.0")
+    mock_version_0_0 = Mock(tag_name="0.0.0.0")
```
Comment (emmyoop) [inline test/unit/github_client/test_get_release.py:29]: Is the 4 zeros intentional?
```suggestion
    mock_version_0_0 = Mock(tag_name="0.0.0")
```

File: test/unit/github_client/test_get_release.py [hunk 2]
```diff
@@ -21,3 +22,31 @@ def test_get_latest_bundle_release_gets_latest_release(monkeypatch, input_versio
...
+                                           mock_version_0_dev, mock_version_0_0])
+
+    monkeypatch.setattr("release_creation.github_client.get_release.get_bundle_repo", lambda: repo_mock)
+    output = get_bundle_release("0.0.0")
+    assert output is None
+
+
+def test_get_bundle_release_returns_release_if_found(monkeypatch):
+    mock_version_1_0_0 = Mock(tag_name="1.0.0", is_draft=False)
+    mock_version_1_1 = Mock(tag_name="1.1.0")
```
Comment (emmyoop) [inline test/unit/github_client/test_get_release.py:43]: nit
```suggestion
    mock_version_1_1_0 = Mock(tag_name="1.1.0")
```

File: test/unit/github_client/test_get_release.py (hunk 1 above)
Comment (emmyoop) [inline test/unit/github_client/test_get_release.py:29]: nit
```suggestion
    mock_version_1_1_0 = Mock(tag_name="1.1.0")
    mock_version_0_0_0 = Mock(tag_name="0.0.0.0")
```

**Review by emmyoop — APPROVED** [review]
Just a few nits but otherwise looks good!

File: test/unit/github_client/test_get_release.py (hunk 1 above)
Comment (colin-rogers-dbt) [reply test/unit/github_client/test_get_release.py:29]: yea I want to test that it won't match if someone creates a bundle with four digits

**Pull request merged**